kdl-client/src/Components/CircledInfoMessage/CircledInfoMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import styles from './CircledInfoMessage.module.scss';
+import cx from 'classnames';
+
+export enum CircledInfoMessageTypes {
```

Comment:
change this to:
```suggestion
export enum CircledInfoMessageTypes {
  SUCCESS = "SUCCESS";
  ERROR = "ERROR";
}
```
See next comment for more info